spec/models/product_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Product, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
no test for the model you need to have those

Reply:
We have implemented tests now